advanced/lighttpd.conf.debian

```diff
@@ -12,6 +12,7 @@
 server.modules = (
 	"mod_access",
 	"mod_accesslog",
+	"mod_auth",
```

Comment:
would `server.modules += ("mod_auth")` inside of an include work and be more modular?